src/witan/sen2/return/person_level/blade_export/csv.clj

```diff
@@ -71,9 +71,10 @@
 
 (def sen2-read-cfg
   "Configuration map for reading SEN2 module 0 \"SEN2\" CSV file into a dataset."
-  {:key-fn       sen2-csv-col-label->name
-   :parser-fn    sen2-parser-fn
-   :dataset-name "sen2"})
+  {:column-allowlist (keys sen2-csv-col-label->name)
```

Comment:
What is the reason for only doing an allowlist rather than just ignoring any extra columns?

Reply:
I decided to limit to the expected columns because other columns are not per spec. and to have additional columns come through nicely you need to augment the metadata for the column names, parser-fn's etc.

After discussion I appreciate that this "only per spec" approach isn't in alignment with the general clojure open ethos of letting things through and may catch us out.

As I've used this for an application I'll keep as is for this PR (so can reproduce the application), but will reverse in future update.